mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -150,33 +150,31 @@ public static Collection<Object[]> parameters() {
 
   private TestTables testTables;
 
-  @Parameter(0)
+  @Parameter(index = 0)
   public FileFormat fileFormat;
 
-  @Parameter(1)
+  @Parameter(index = 1)
   public String executionEngine;
 
-  @Parameter(2)
+  @Parameter(index = 2)
   public TestTables.TestTableType testTableType;
 
-  @Parameter(3)
+  @Parameter(index = 3)
   public boolean isVectorized;
 
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
+  @TempDir public Path temp;
```

Comment:
```suggestion
  @TempDir private Path temp;
```
same for all the other parameters